### DISCUSSION TITLE
why can't i snipe these codes

### DISCUSSION THREAD

**Discussion by notquite28** [GiorgioBrux/nitro-sniper-enhanced#170] [Q&A]
Tried to redeem code [WfvnvNM8HrcTQgyRGPFxQVae] but the gift type cannot be used with an existing Nitro.
![image](https://user-images.githubusercontent.com/43907678/139461349-16e3a7df-5801-4ce0-8083-0398a8b9add2.png)
is this some different nitro type?

**Comment by GiorgioBrux** [comment]
No idea. We just try to redeem the gift and then print basically what discord tells us. It doesn't give the reason why it's not compatible, so I don't have a clue.
